css/media.css

```diff
@@ -0,0 +1,3 @@
+@media (max-widht: 767px) {
+  
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/teslaproject/pullRequest?prid=5566861)